src/lib/api/services/ai-chat/ai-chat.service.ts

```
@@ -1,15 +1,26 @@
 'use client';
 
-import { useMutation, useQuery } from '@tanstack/react-query';
+import { type InfiniteData, useInfiniteQuery, useMutation } from '@tanstack/react-query';
+import { type SessionListData } from './ai-chat.type';
 import { QUERY_KEY } from '@/constants';
 import { createSession, getSessions } from './ai-chat.client';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)